### ISSUE TITLE
Type error: OrbitControls.listenToKeyEvents() does not accept global Window object

### ISSUE THREAD

**Issue by svdHero** [three-types/three-ts-types#210]
### Environment:

-   `three` version: 0.138.3
-   `node` version: 14.17.5
-   `npm` version: 6.14.14

My application is written with Create React App and Typescript.

### Problem description:

@Mugen87 explained in [this post](https://github.com/mrdoob/three.js/issues/4327#issuecomment-855203187) that in order to enable key events for `OrbitControls`, one has to add this line
```ts
controls.listenToKeyEvents(window)
```
However, when I do this, I get the following type error:
```
Argument of type 'Window & typeof globalThis' is not assignable to parameter of type 'HTMLElement'.
```
As a consequence, it is not possible to add keyboard support for `OrbitControls` to my application at the moment.

### Suggested solution:

Fix the type signature for the method
```ts
listenToKeyEvents(domElement: HTMLElement): void;
```
in `OrbitControls.d.ts` such that it accepts the global `Window` object.

**Comment by joshuaellis** [comment]
Will be resolved in `0.140.0`

**Comment by svdHero** [comment]
@joshuaellis Thank you so much!
